components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java

```diff
@@ -478,6 +489,34 @@ && isStepHasMultiOption(context)
         }
     }
 
+    private boolean checkIfApplicationEnabled(HttpServletRequest request, AuthenticationContext context)
```

Comment:
```suggestion
    private boolean isApplicationEnabled(HttpServletRequest request, AuthenticationContext context)
```

The suggestion make it easier to understand code in this private method usages. Otherwise it looks like a void method